maps/site53/structures/signs.dm

```diff
@@ -47,8 +47,8 @@
 	icon_state = "solgovseal"
 
 /obj/structure/sign/double/solgovflag
-	name = "Sol Central Government Flag"
-	desc = "The flag of the Sol Central Government, a symbol of many things to many people."
+	name = "United Nations Global Occult Coalition Flag"
```

Comment:
Maybe make this just the UNs flag?

also resprite it, of course.

Reply:
The SolGov flag is already really similar to UN flag. You can spawn it in game if you'd like to see. Also I don't have any experience spiriting.

Reply:
Just use the byonds built-in spriting tool, we already have a shit ton of codersprites whats one more

Reply:
just needs a resprite

Reply:
![image](https://github.com/Foundation-19/Foundation-19/assets/77739066/7f2e511f-08d9-43e6-88e5-e2cdcce20ce2)